examples/prism-pascaligo.html

```diff
@@ -0,0 +1,63 @@
+<h2>Comments</h2>
+<pre><code>// Single line comment
+(* Multi-line
+comment *)</code></pre>
+
+<h2>Strings</h2>
+<pre><code>"foo \"bar\" baz";
+'foo \'bar\' baz';</code></pre>
+
+<h2>Numbers</h2>
+<pre><code>123
+123.456
+-123.456
+1e-23
+123.456E789
+0xaf
+0xAF
+</code></pre>
+
+<h2>Functions</h2>
+<pre><code>foo()
+Bar()
+_456()
+</code></pre>
+
+<h2>Full Example</h2>
+<pre><code>
+    function pop (const h : heap) : (heap * heap_element * nat) is
```

Comment:
I really like the example but its indentation is kind of awkward. Sometimes one space is used, sometimes two. If there is no preferred indentation style set by the language (or its community) then please opt for tabs.
Also, the... well... global indentation of 4 spaces should be removed.